### ISSUE TITLE
app breaks when turning knob on minimum_patch.json 

### ISSUE THREAD

**Issue by michaelpalumbo** [worldmaking/mischmasch#189]
```shell
D:\github\mischmasch\Components\Patch\Patch.js:199
        for(let i = 0; i < this.document[opID].inputs.length; i++){

   ^

TypeError: Cannot read property 'inputs' of 
undefined
    at Patch.update (D:\github\mischmasch\Components\Patch\Patch.js:199:48)
    at Object.updateHandStateMachine (D:\github\mischmasch\app.js:601:13)
    at Object.updateStateMachines (D:\github\mischmasch\app.js:444:37)
    at Immediate.animate (D:\github\mischmasch\app.js:2190:5)
    at processImmediate (internal/timers.js:461:21)
```

**Comment by michaelpalumbo** [comment]
this is likely due to the change in the op structure. let's see...

**Comment by michaelpalumbo** [comment]
update: minimum_patch.json had underscores in its uuids, which breaks a fair bit of other elements of mischmasch. This was resolved in #184 which just removes all hyphens from uuid4v() at creation.
